lib/collective.lib.js

```diff
@@ -49,6 +49,11 @@ export const getCollectiveMainTag = (hostCollectiveId = null, tags = [], type, s
   return CollectiveCategory.COLLECTIVE;
 };
 
+export const expenseSubmissionAllowed = (collective, user) => {
+  const member = user?.memberOf.filter(member => member.collective.slug === collective.slug);
```

Comment:
Maybe we want to filter the kind of members. Implemented this way, seems financial contributors will be able to submit expenses.

That might be actually fine for now.

Reply:
This won't play well with https://github.com/opencollective/opencollective/issues/1445 (anyone will be able to create a member by "subscribing" to the collective). A small filter on the role might be beneficial, but I agree we can keep things like this for now.

Reply:
So I'll keep this for now. 👍🏽